### PR TITLE
create.hub.flow

### PR DESCRIPTION
**Pull request by nodatall**

### CODE REVIEW

File: organization_types_categories.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+'use strict';
```
Comment (deadlyicon) [inline organization_types_categories.js:1]: i think this is only needed by the OrganizationCreatePage component

**Review by deadlyicon — CHANGES_REQUESTED** [review]

File: organization_types_categories.js (hunk 1 above)
Comment (nodatall) [reply organization_types_categories.js:1]: nope, b-api has to check to make sure purpose / type combination are valid

File: test/organizationPurposes.spec.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const ORGANIZATION_TYPES_LABELS = require('../organization_types_labels');
+const ORGANIZATION_TYPES_BY_PURPOSE = require('../organization_types_by_purpose');
+
+describe('organization categories', function(){
```
Comment (deadlyicon) [inline test/organizationPurposes.spec.js:6]: 👉

File: test/organizationPurposes.spec.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  it('all types in category should be valid', function(){
```
Comment (deadlyicon) [inline test/organizationPurposes.spec.js:7]: 👉

**Review by deadlyicon — CHANGES_REQUESTED** [review]

File: test/organizationPurposes.spec.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+'use strict';
```
Comment (deadlyicon) [inline test/organizationPurposes.spec.js:1]: why does this test have like a random name? why not `test/organization_types_by_purpose.spec.js`?

**Review by deadlyicon — CHANGES_REQUESTED** [review]

File: test/organizationPurposes.spec.js (hunk 4 above)
Comment (nodatall) [reply test/organizationPurposes.spec.js]: huh yeah, that was a bad name. renamed.

File: test/organization_types_by_purpose.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+describe('organization purposes', function(){
+  it('all types in purpose should be valid', function(){
+    Object.values(ORGANIZATION_TYPES_BY_PURPOSE).forEach(types => {
+      types.forEach(type => {
+        expect(ORGANIZATION_TYPES_LABELS[type]).to.exist;
+      });
+    });
+  });
+});
```
Comment (deadlyicon) [inline test/organization_types_by_purpose.js:14]:
```suggestion
describe('ORGANIZATION_TYPES_BY_PURPOSE', function(){
  it('should only contain valid types', function(){
    Object.values(ORGANIZATION_TYPES_BY_PURPOSE).forEach(types => {
      types.forEach(type => {
        expect(ORGANIZATION_TYPES_LABELS[type]).to.exist;
      });
    });
  });
});
```

**Review by deadlyicon — CHANGES_REQUESTED** [review]

File: test/organization_types_by_purpose.js (hunk 5 above)
Comment (nodatall) [reply test/organization_types_by_purpose.js]: done

**Review by deadlyicon — APPROVED** [review]

**Pull request merged**